scripts/submit-analytics.js

```diff
@@ -259,7 +239,25 @@ async function submitAnalytics(dryRun) {
     return;
   }
 
-  await Promise.all([dispatchAnalytics(payload)]);
+  return new Promise((resolve, reject) => {
+    // node 19 turns on keep-alive by default and it will make the https.get() to hang
+    // https://github.com/realm/realm-js/issues/5136
+    https.globalAgent = new https.Agent({ keepAlive: false });
+    const requestUrl = getAnalyticsRequestUrl(payload);
+
+    https
+      .get(requestUrl, (res) => {
```

Comment:
Rather than overriding the global agent here, why not pass the custom agent to the `get` request options:

```suggestion
    const requestUrl = getAnalyticsRequestUrl(payload);

    https
      .get(requestUrl, { agent:  new https.Agent({ keepAlive: false }) }, (res) => {
```